backend/src/main/java/ee/tlu/forum/service/UserServiceInterface.java

```diff
@@ -6,17 +6,18 @@
 import java.util.List;
 
 public interface UserServiceInterface{
-    User editUser(User user);
+    User editUser(User user, String token);
     User registerUser(User user);
-    Role saveRole(Role role);
-    void addRoleToUser(String username, String roleName);
+    Role saveRole(Role role, String token);
+    void addRoleToUser(String username, String roleName, String token);
     List<User> getUsers();
-    User getUserById(Long id);
-    User getUserByUsername(String username);
+    User getUserById(Long id, String token);
+    User getUserByUsername(String username, String token);
+    User getUserByUsernameAuthorized(String username);
```

Comment:
Should these be the other way around? Authorized with token and regular without?

Reply:
Authorized is completely removed in another PR 🎁